docs/advanced/global_configuration.md

```diff
@@ -29,4 +32,61 @@ change_ps1 = true
 # security risk and should only be used for testing purposes or internal networks.
 # You can override this from the CLI with `--tls-no-verify`.
 tls_no_verify = false
+
+[mirrors]
+# redirect all requests for conda-forge to the prefix.dev mirror
+"https://conda.anaconda.org/conda-forge": [
+    "https://prefix.dev/conda-forge"
+]
+
+# redirect all requests for bioconda to one of the three listed mirrors
+# Note: for repodata we try the first mirror first.
+"https://conda.anaconda.org/bioconda": [
+    "https://conda.anaconda.org/bioconda",
+    # OCI registries are also supported
+    "oci://ghcr.io/channel-mirrors/bioconda",
+    "https://prefix.dev/bioconda",
+]
 ```
+
+## Mirror configuration
+
+You can configure mirrors for conda channels. We expect that mirrors are exact
+copies of the original channel. The implementation will look for the mirror key
+(a URL) in the `mirrors` section of the configuration file and replace the
+original URL with the mirror URL.
+
+To also include the original URL, you have to repeat it in the list of mirrors.
+
+The mirrors are prioritized based on the order of the list. We attempt to fetch
+the repodata (the most important file) from the first mirror in the list. The
+repodata contains all the SHA256 hashes of the individual packages, so it is
+important to get this file from a trusted source.
+
+You can also specify mirrors for an entire "host", e.g.
+
+```toml
+[mirrors]
+"https://conda.anaconda.org": [
+    "https://prefix.dev/"
+]
+```
+
+This will forward all request to channels on anaconda.org to prefix.dev.
+Channels that are not currently mirrored on prefix.dev will fail in the above example.
+
+### OCI Mirrors
+
+You can also specify mirrors on the OCI registry. There is a public mirror on
+the Github container registry (ghcr.io) that is maintained by the conda-forge
+team. You can use it like this:
+
+```toml
+[mirrors]
+"https://conda.anaconda.org/conda-forge": [
+    "oci://ghcr.io/channel-mirrors/conda-forge"
+]
```

Comment:
```suggestion
"https://conda.anaconda.org/conda-forge" = [
    "oci://ghcr.io/channel-mirrors/conda-forge"
]
```

Reply:
This also give the same error with the channel repodata, I guess you are missing the name of the channel.

Reply:
Fixed by always forcibly adding a trailing `/` - good catch :)